sycl/include/sycl/ext/intel/fpga_dsp_control.hpp

```diff
@@ -0,0 +1,79 @@
+//==------------ fpga_dsp_control.hpp --- SYCL FPGA DSP Control ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#pragma once
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
```

Comment:
The spec is in the final draft state. isn't it resonable to move the extension to experimental namespace?

Reply:
Which spec are you referring to? I thought we can release the header since the SPIR-V support is published.

Reply:
https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/INTEL/SPV_INTEL_fpga_dsp_control.asciidoc

Reply:
All published specs under extensions/INTEL are documented as "final draft" state AFAIK. I don't think it's a concern for delivering the header.